### DISCUSSION TITLE
Merge Symbols, Merge Layers, Keyboard Support!

### DISCUSSION THREAD

**Discussion by CartoonMonkeyStudio** [keiwando/toonsquid#115] [Ideas]
It would be great to be able to merge layers into one.. merge symbols into one symbol, and add keyboard support for undo, flipping through frames, layers etc…

**Comment by keiwando** [comment]
By merging symbols do you essentially mean being able to add the layers of one symbol on top of the layers from another symbol?

There are already some keyboard shortcuts. You can for example use the `,` and `.` keys to move the playhead backwards and forwards through the timeline.
https://toonsquid.com/handbook/interface/shortcuts/
However, I just noticed that the Undo and Redo shortcuts don't seem to be working at the moment. I will look into that!

Thanks for the suggestions!
